.circleci/config.yml

```diff
@@ -451,7 +451,10 @@ jobs:
             apt-get -qq install jq -y
             pip install --quiet awscli
             ./aws.sh \
+              --API_AWS_ACCESS_KEY_ID "${<< parameters.env >>_API_AWS_ACCESS_KEY_ID" \
+              --API_AWS_SECRET_ACCESS_KEY "${<< parameters.env >>_API_AWS_SECRET_ACCESS_KEY" \
               --API_DATABASE_URL "${<< parameters.env >>_API_DATABASE_URL}" \
+              --API_FILE_S3_BUCKET "${<< parameters.env >>_API_FILE_S3_BUCKET" \
```

Comment:
These are the credentials and the S3 bucket to write files into. Staging/production save to S3.